tests/integration/telemetry/stackdriver/testdata/traffic_assertion.json.tmpl

```diff
@@ -13,7 +13,7 @@
         "canonical_service": "srv",
         "canonical_revision": "v1"
     },
-    "protocol": "PROTOCOL_GRPC",
+    "protocol": "PROTOCOL_HTTP",
```

Comment:
Why is this changed?

Reply:
+1

Reply:
edge reporting is cached and keyed by peers. protocol is not part of the cache key. so, if whatever test ordering happens in the environment with this test causes HTTP traffic to be sent first in some cases.

i'd suggest "zero"-ing out the protocol bits in the validation, so that there is a match regardless of actual protocol. 

we could "fix" edge reporting, but it is going away within a release cycle most likely, so we can be blunt about the fix.